### PR TITLE
Add plugin list command to show installed plugins

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #199

### CODE REVIEW

**Comment by mohammadranjbarz** [comment]
For testing it you just need to enter this
`ts-node packages/cli/bin/cannon.ts plugin list`

Screen Shot 1402-03-24 at 14 33 47

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -503,6 +505,16 @@ program
...
 const pluginCmd = program.command('plugin').description('Manage Cannon plug-in modules');
 
+pluginCmd
+  .command('list')
+  .description('List all installed plugins')
+  .action(async function () {
+    console.log(green(bold('\n=============== Installed Plugins ===============')));
+    const installedPlugins = await listInstalledPlugins();
+    installedPlugins.forEach((plugin) => console.log(yellow(plugin)));
+    console.log('Complete!');
```
Comment (noahlitvin) [inline packages/cli/src/index.ts:515]: I think we can probably remove this line, otherwise lgtm!

Comment (mohammadranjbarz) [reply packages/cli/src/index.ts:515]: @noahlitvin  Which line do you mean( you have selected four lines), I assume you mean `console.log('Complete!');` Am I right?

Comment (dbeal-eth) [reply packages/cli/src/index.ts:515]: yes that line

**Review by dbeal-eth — APPROVED** [review]

**Pull request merged**